app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,5 @@
     <string name="app_name">ParkingSystem</string>
     <string name="text_main_activity_title">Welcome to Parking System</string>
     <string name="button_main_activity_total_parking_lots">Select parking lots</string>
+    <string name="toast_main_activity_total_parking_lots">Total of parking lots: </string>
```

Comment:
here the resource should be 
Total of parking lots:  %1$d
so you can use
context.getString(R.string.toast_main_activity_total_parking_lots, parkingLots)

Reply:
Done!